app/containers/UnlockEtherContainer.tsx

```diff
@@ -0,0 +1,200 @@
+  }
+
+  public render() {
+    if (this.props.isLoading) {
+      return <LoadingIndicator />;
+    }
+
+    return (
+      <div>
+        <h1>Unlock your funds</h1>
+        <p>You can unlock your funds anytime you want.</p>
+        <p>
+          You need to give back all neumarks rewarded to you. If you unlock before unlock period is
+          over you will be penalized with 10% value of your funds.
+        </p>
+        <div className={styles.section}>
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Your wallet address:</div>
+            <div className={styles.value}>
+              {this.props.params.address}
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Your NEU balance: </div>
+            <div className={styles.value}>
+              <MoneyComponent value={this.props.neumarkBalance} tokenType={TokenType.NEU} />
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Ether locked: </div>
+            <div className={styles.value}>
+              <MoneyComponent value={this.props.lockedAmountEth} tokenType={TokenType.ETHER} />
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>NEU needed to unlock ether funds: </div>
+            <div className={styles.value}>
+              <MoneyComponent
+                value={this.props.neumarkNeededToUnlockEth}
+                tokenType={TokenType.NEU}
+              />
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Free unlock date: </div>
+            <div className={styles.value}>
+              {this.props.unlockDateEth.format("YYYY-MM-DD")}
+            </div>
+          </div>
+        </div>
+        {this.renderSteps()}
+      </div>
+    );
+  }
+
+  private renderSteps() {
+    const neumarkNeededToUnlockAsBI = new BigNumber.BigNumber(this.props.neumarkNeededToUnlockEth);
+    const neumarkBalanceAsBI = new BigNumber.BigNumber(this.props.neumarkBalance);
+
+    const withdrawPossible = neumarkBalanceAsBI.greaterThanOrEqualTo(neumarkNeededToUnlockAsBI);
+    const willBePenalized = moment().isBefore(this.props.unlockDateEth);
+
+    if (!withdrawPossible) {
+      return (
+        <div>
+          <Alert bsStyle="danger">
+            You need{" "}
+            <MoneyComponent
+              value={neumarkNeededToUnlockAsBI.sub(neumarkBalanceAsBI)}
+              tokenType={TokenType.NEU}
+            />{" "}
+            Neumarks more to unlock your account.
+          </Alert>
+        </div>
+      );
+    }
+
+    return (
+      <div className={styles.section}>
+        {willBePenalized
+          ? <Alert bsStyle="danger">
+              Withdrawing funds now will result in penalty of 10% of your funds!
```

Comment:
Make it a checkbox!
`I confirm that by following unlock procedure I agree to return XXX NEU and to pay XXX.XXXX unlock fee to remaining NEU holders`